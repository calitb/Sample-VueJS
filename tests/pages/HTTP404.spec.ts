import { shallowMount } from "@vue/test-utils";
import HTTP404 from "@/pages/HTTP404.vue";

describe("HTTP404 page", () => {
  describe("Default", () => {
    const wrapper = shallowMount(HTTP404);

    it("should render the component", () => {
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
