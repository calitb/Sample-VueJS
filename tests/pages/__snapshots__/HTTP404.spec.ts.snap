// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HTTP404 page Default should render the component 1`] = `
<div>
  
  NO EXISTE

</div>
`;
